README.md

```diff
@@ -34,7 +35,7 @@ This Go package is automatically generated by the [Swagger Codegen](https://gith
 - Package version: 3.4.2
 - Build package: io.swagger.codegen.v3.generators.go.GoClientCodegen
 
-## Requirements.
+## Requirements
 
 Go Lang 1.17 or higher
```

Comment:
@TheUnderScorer @ilfa Does it make sense to define some kind of support policy here?

Reply:
We can add a note that we'll support two latest major versions of GO. WDYT @ilfa ?

Reply:
We can say that we keep Microsoft support policy (2 latest major versions as you mentioned) and provide the [link](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core) to it

Reply:
I think that you meant this for Dotnet @ilfa 😁. What about GO?

Reply:
Oh no!
But it is fair for a golang too, we just need [another URL](https://go.dev/doc/devel/release#:~:text=checkout%20goX.Y.Z-,Release%20Policy,until%20the%20Go%201.8%20release.)

Reply:
Done!